engine/gameobject/src/gameobject/comp_anim.cpp

```diff
@@ -707,7 +707,9 @@ namespace dmGameObject
         {
             if (INVALID_INDEX == next)
             {
-                world->m_ListenerInstanceToIndex.Erase((uintptr_t)anim->m_Userdata1);
+                uint16_t* p = world->m_ListenerInstanceToIndex.Get((uintptr_t)anim->m_Userdata1);
+                if (p != 0)
+                    world->m_ListenerInstanceToIndex.Erase((uintptr_t)anim->m_Userdata1);
```

Comment:
Making sure the key exists, before deleting it (the hash table asserts otherwise).

This only happens when cleaning up an animation that failed to register due to reaching max number of go animate.